source/_posts/2020-10-21-supervisor-249.markdown

```diff
@@ -0,0 +1,72 @@
+---
+title: "Supervisor 249: Resolution center, Stability & Private container registries"
+description: "Resolution center, Stability & Private container registries"
+date: 2020-10-21 00:00:00
+date_formatted: "October 21, 2020"
+comments: true
+author: Joakim Sørensen
+author_twitter: ludeeus
+categories: Release-Notes
+og_image: /images/blog/2020-10-21-supervisor-249/social.png
+---
+
```

Comment:
We should assume that people coming to this post don't know what the supervisor is. We should write a generic introduction of the supervisor and put that in cursive at the top.

Maybe something like this (not happy with it, I lack inspiration atm)

```suggestion

_The Supervisor is in charge of the Home Assistant Operating System and allows users to easily manage it via the Home Assistant User interface._
```

Reply:
This makes it seem it's only the OS.

`s/the Home Assistant Operating System/managing your system/`

Reply:
```suggestion
The Supervisor is shipped with our Home Assistant Operating System and controls all hardware and software on your device. Bringing an effortless experience in managing, upgrading and maintaining your Home Assistant instance. It provides single-click upgrades for the Operating System and Home Assistant Core, but also provides easy to use and install add-ons and allows users to easily manage it via the Home Assistant User interface. It is what allows you to focus on what really matters; automating your home.
```